deps.sh

```diff
@@ -2,6 +2,9 @@
 # Add new direct and transitive dependencies as submodules.
 # Note: caller must validate/curate/commit changes to .gitmodules and 'vendor/'
 #
+echo "Note: caller must validate/curate/commit changes to .gitmodules and 'vendor/'"
```

Comment:
So if it doesn't add the dependency correctly TravisCI will break.  @jasonbs10 can probably answer this question better.  He asked me to add the echo so the comment actually gets seen when the script is ran.